src/main/java/org/kaazing/netx/ws/WsURLConnection.java

```diff
@@ -121,8 +120,16 @@ protected WsURLConnection(URL url) {
     }
 
     /**
-     * Disconnects with the server. This is a blocking call that returns only
-     * when the shutdown is complete.
+     * Adds the specified {@link WebSocketExtension} object to the list of enabled extensions. The extension's
+     * <code>toString()</code> method must return RFC-3864 formatted string so that it can be negotiated using the
+     * <i>Sec-Websocket-Extensions</i> header during the opening handshake.
+     *
+     * @param extension {@link WebSocketExtension} with <code>toString()</code> method that returns RFC-3864 formatted string
+     */
+    public abstract void addEnabledExtension(WebSocketExtension extension);
```

Comment:
I changing the following signature:

`private void addEnabledExtension(WebSocketExtension extension)` 

to

`private void addEnabledExtensions(String... extensions)`